### PR TITLE
DTSPO-16508 - upgraded non prod traefik

### PR DESCRIPTION
**Pull request by cpareek**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/DTSPO-16508

### Change description ###
Upgraded traefik to v26, followed same configuration done as on sbox

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines

### CODE REVIEW

**Review by thomast1906 — APPROVED** [review]

File: apps/admin/traefik2/ptl-intsvc/00.yaml [hunk 1]
```diff
@@ -4,6 +4,15 @@ metadata:
   name: traefik2
   namespace: admin
 spec:
+  chart:
```
Comment (endakelly) [inline apps/admin/traefik2/ptl-intsvc/00.yaml:7]: I would do ptl in sds first just to be sure of Jenkins working. Should be fine but best to keep it consistent

Comment (cpareek) [reply apps/admin/traefik2/ptl-intsvc/00.yaml:7]: ah -sure, I was thinking this would be different then ptl  environment as it said ptl-intsvc :)

Comment (endakelly) [reply apps/admin/traefik2/ptl-intsvc/00.yaml:7]: yeah, sadly they're not the same across both repos but they are the same use case

Comment (cpareek) [reply apps/admin/traefik2/ptl-intsvc/00.yaml]: sure

**Review by endakelly — APPROVED** [review]

**Review by endakelly — APPROVED** [review]

**Pull request merged**